Standards/scs-0300-v1-requirements-for-sso-identity-federation.md

```diff
@@ -213,6 +215,19 @@ this should allow customers to manage settings like these in their own IAM.
   - How would we allow SCS operators to choose?
 - Do we need some kind of SWOT analysis to come to a decision?
 
+## Decision
+
+SCS wants to make use of an IdP as part of the reference implementation.
+To move forward with topics of configuration and mapping of roles in a
+OAuth federated scenario as well as questions of token lifecycles etc across
+the federation stack it makes sense to fokus on one IdP implementation at a
+given time.
+
+Since Keycloak currently supports the OAuth 2.0 grants that SCS wants to make
+use of (e.g. Device Authorization Grant) and offers additional flexibility
+e.g. by allowing LDAP to be used as backend, the projects current choice
```

Comment:
```suggestion
e.g. by allowing LDAP to be used as backend, the project's current choice
```

Reply:
Also, Zitadel [supports LDAP](https://zitadel.com/docs/guides/integrate/identity-providers/ldap) as backend now, as well.
So this specific feature may be dropped as advantage of Keycloak, here.